### DISCUSSION TITLE
Request for Extension of Tool Submission Deadline

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#34] [General]
Hello, 

Would it be possible to extend the tool submission deadline for the SDC Testing Competition (SBFT 2025, Ottawa, Canada  track) ?

Thank you for your time, and have a great day.

**Comment by ChristianBirchler** [comment]
Hi @vatozZ, we extended the deadline.

Happy coding!

**Reply by ali-ihsan-gullu** [reply]
Thank you so much : )
